recipes/benchmark/all/conanfile.py

```diff
@@ -44,9 +44,10 @@ def layout(self):
         cmake_layout(self, src_folder="src")
 
     def validate(self):
-        if self.info.settings.compiler == "Visual Studio" and Version(self.info.settings.compiler.version) <= "12":
+        check_min_vs(self, "140")
```

Comment:
```suggestion
        check_min_vs(self, "190")
```

Assuming that the check implies that it works with Visual Studio 14.0 and above (there was no version 13), this would mean the minimum is `msvc` is version 190 (handy [table](https://en.wikipedia.org/wiki/Microsoft_Visual_C%2B%2B)). 
No need to retain the check against the "Visual Studio" compiler, `check_min_vs` will perform the correct comparison already :D